components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -532,6 +548,10 @@ void RRTMGPRadiation::initialize_impl(const RunType /* run_type */) {
     auto co_vmr = get_field_out("co_volume_mix_ratio").get_view<Real**>();
     Kokkos::deep_copy(co_vmr, m_params.get<double>("covmr", 1.0e-7));
   }
+
+  // Ensure rad_heating_pdel is recognized as initialized by the driver
```

Comment:
Is this needed because of the way rad_heating_pdel is explicitly added as an internal field above? Why does rad_heating_pdel need this, but the surface fluxes do not?

Reply:
Uhm, good point. The reason is related to IO: when IO runs, it checks that the fields have a valid timestamp (otherwise it assumes we're writing garbage).

I think we prob don't need to set the timestamp on pdel right here, but we would need to set the timestamp at runtime then. For surf fluxes, since they are "Computed", the base class `AtmosphereProcess` takes care of updating the time stamp at the end of `run`, so whenever the restart happens, they would have a valid timestamp. For internal fields, we need to set a valid timestamp _at some point_. I figured "let's set t0 at init, and not worry about it ever again", but maybe, for simmetry, we should set the timestamp during run_impl...